config/Migrations/20160716113123_Initial.php

```diff
@@ -1,51 +1,53 @@
 <?php
-use Migrations\AbstractMigration;
 
-class Initial extends AbstractMigration
-{
+  use Migrations\AbstractMigration;
+
+  class Initial extends AbstractMigration {
+
```

Comment:
CakePHP has its own standard: http://book.cakephp.org/3.0/en/intro/conventions.html

You can validate your code using: https://github.com/cakephp/cakephp-codesniffer

It might be a good idea to follow those standards and conventions... Probably there is a plugin for Netbeans to auto format to CakePHP's conventions, else you should do it manually (http://blog.bobbyallen.me/2013/03/24/configuring-netbeans-for-psr-1-and-psr-2-coding-guidelines/).